plugins/module_utils/copy.py

```diff
@@ -79,55 +123,72 @@ def __init__(self, module, client):
         super(K8SCopyFromPod, self).__init__(module, client)
         self.is_remote_path_dir = None
         self.files_to_copy = list()
+        self._shellname = None
+
+    @property
+    def pod_shell(self):
+        if self._shellname is None:
+            for s in ("/bin/sh", "/bin/bash"):
```

Comment:
I'm pretty sure that if you've got /bin/bash, you also get /bin/sh. So it safes to default on the last one.